config/packages/doctrine.yaml

```diff
@@ -7,18 +7,17 @@ parameters:
 
 doctrine:
     dbal:
+        url: '%env(resolve:DATABASE_URL)%'
+
         # configure these for your database server
         driver: 'pdo_mysql'
         server_version: '10.2.19-mariadb'
         charset: utf8mb4
         default_table_options:
-            charset: utf8mb4
             collate: utf8mb4_unicode_ci
 
-        url: '%env(resolve:DATABASE_URL)%'
     orm:
         auto_generate_proxy_classes: true
-        metadata_cache_driver: apcu
```

Comment:
We have ACPU in the setup...

Reply:
Config key is deprecated: https://symfony.com/doc/4.4/reference/configuration/doctrine.html#caching-drivers

>All the Doctrine caching types are deprecated since Symfony 4.4 and won't be available in Symfony 5.0 and higher. Replace them with either type: service or type: pool and use any of the cache pools/services defined with Symfony Cache.

Reply:
Also: https://stackoverflow.com/questions/64741823/symfony-4-4-metadata-cache-driver-configuration-key-deprecation-notice